src/components/item/Item.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+
+interface ItemProps extends React.HTMLAttributes<HTMLInputElement> {
+  type?: 'radio' | 'checkbox';
+  name?: string;
+  label?: string;
+  emjCode?: string | undefined;
```

Comment:
```suggestion
  emjCode?: string;
```
이렇게 작성해도 동일하게 타입 추론이 될 거 같아요 ~!